.dockerignore

```diff
@@ -0,0 +1,2 @@
+tutorials
+third_party
```

Comment:
I don't think those should be ignored, someone might want to run the docker image to test the tutorials which he expect to be there, third_party libraries also doesn't need to be downloaded if they are already on the host machine. Some example directories that might need to be ignored are build, bin, dist, lib, and wheels which might be generated in the working directory when installing and building the library.

Reply:
Ah ok yeah, I misunderstood what third party was, I will remove that line.

As for tutorials, they are data, image, and jupyter notebook files from the looks of it. However they have no way to run jupyter  files in your docker containers, nor does the container expose a port for users to interact with the jupyter server if it was running, so they could play around with these tutorials even if the files were included. 
I will remove that line if you still want the file in, but I don't think its reasonable to expect an average user to be able to modify the docker container to be able to run jupyter notebooks since the container does not support it, and they are unnecessary for everyone else to my understanding.

Reply:
True that we doesn't install juputer-notebook in the current images, maybe we should ignore them and build a specific image for that. Then we can leave it as is, but can you add [build, bin, dist, lib, wheels] also, those are the one I know about which are generated when I'm building and playing with the package, I don't want them to be included in the docker image.

Reply:
Yeah no problem will add them now.

Reply:
also, please merge master into your branch so that I can merge it back to master

Reply:
done

Reply:
Thank you!